recipes/zen3geo/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "zen3geo" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/zen3geo-{{ version }}.tar.gz
+  sha256: 09e84306809d03899155510f75dff83023e301c95bb0835b7bf5c34e58c72720
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - poetry >=1.2.0rc2
+    - python >=3.8,<3.12
+  run:
+    - python >=3.8,<3.12
+    - rioxarray >=0.10.0
+    - torchdata >=0.4.0
+
+test:
+  imports:
+    - zen3geo
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: None
```

Comment:
Can we add something here?

Reply:
Oh yes, I can add in the readthedocs page, will do it in the feedstock repo.